cms_helper.py

```diff
@@ -23,6 +23,7 @@ def gettext(s):
         'permissions': 10,
     },
     'ROOT_URLCONF': 'tests.test_utils.urls',
+    'MIGRATION_MODULES': 'DisableMigrations()',
```

Comment:
This can't be a string. It needs to be instantiating the class itself. 

Also please put the following code in `cms_helper.py` as well. 
```
class DisableMigrations(object):
    def __contains__(self, item):
        return True

    def __getitem__(self, item):
        return None
```
```suggestion
    'MIGRATION_MODULES': DisableMigrations(),
```

Reply:
but this DisableMigrations() needs to be defined or imported from somewhere?